_posts/2021-07-13-part 21.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+author: Anthony
+---
+# Anthony's Blog
+Anthony's Adventures Around ISD!
+---
+
+title: Day 21!
+date:  2021-07-13
+---
+
+Hi. It's me. Again. Today I started on my powerpoint to show my time at the end of summer job! I can't believe it;s almost over. I've been having a lot of fun at this job and it's been teaching me a lot. I'm very very glad I got this chance to work here and I always appreciate the people here teaching me along the way. I hope I can finish up with all the knowledge that will be helpful for college. My shift is coming to a end. Anthony out! -Anthony
```

Comment:
Looks good, but there are a few typos. 

- I can't believe **it's** almost over.
- job**,** and it's been teaching me a lot
- work here**,** and I always